main/mimic_main.c

```diff
@@ -78,7 +78,7 @@ void sigint_handler(int signum)
 static void mimic_version()
 {
     printf("  Carnegie Mellon University, Copyright (c) 1999-2011, all rights reserved\n");
-    printf("  mimic developers, Copyright (c) 2016, all rights reserved\n");
+    printf("  mimic developers, Copyright (c) 2016-2020, all rights reserved\n");
     printf("  version: %s-%s (%s)\n",
            PACKAGE_NAME, PACKAGE_VERSION, PACKAGE_URL);
```

Comment:
The PACKAGE_URL is not being set when I build. Should this be set to something? (or maybe just removed?)

Reply:
It doesn't show for me either... Looking at the url and the [ac_init documentation](https://www.gnu.org/software/autoconf/manual/autoconf-2.67/html_node/Initializing-configure.html) it seems like it's the PACKAGE_BUGREPORT we add. But we don't use it.

I say remove it. The bugreport url might be good to have in the help but feels misplaced when checking the version.